tools/update_schema.ps1

```diff
@@ -0,0 +1,72 @@
+<#
+.DESCRIPTION
+Updates aspnetcore_schema.xml to the latest version.
+Updates aspnetcore_schema.xml to the latest version.
+Requires admin privileges.
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param()
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 1
+
+$ancmSchemaFiles = @(
+    "aspnetcore_schema.xml",
+    "aspnetcore_schema_v2.xml"
+)
+
+$ancmSchemaFileLocations = @(
+    @(Resolve-Path "$PSScriptRoot\..\src\AspNetCoreModuleV1\AspNetCore\aspnetcore_schema.xml"),
```

Comment:
@jkotalik I'm assuming these file paths are meant to be for IISIntegration, not the ServerTests repo, right? Any reason I shouldn't just copy that file into ServerTests?

Reply:
Yeah let's go ahead and copy.